yaml/xpu_functions.yaml

```diff
@@ -185,6 +185,9 @@ supported:
   - clamp.out
   - clamp_min.out
   - clamp_max.out
+  - clamp.Tensor_out
+  - clamp_min.Tensor_out
```

Comment:
We need add all variants.
<img width="400" alt="image" src="https://github.com/intel/torch-xpu-ops/assets/28250760/6e004265-a6ec-4519-8ffb-5042c352e7b0">

Reply:
done